R/spec.R

```diff
@@ -119,7 +119,11 @@ spec_reason <- function(call, signaller) {
   )
 }
 
-spec_package <- function(env, signaller) {
+spec_pkg <- function(pkg, env, signaller) {
+  if (!is_null(pkg) || is.null(env)) {
```

Comment:
```suggestion
  if (!is.null(pkg) || is.null(env)) {
```

Looks funny to have two different options side by side

Reply:
I use the underscore variant in my packages :)

```suggestion
  if (!is_null(pkg) || is_null(env)) {
```